install/kubernetes/helm/istio/charts/tracing/templates/service.yaml

```diff
@@ -38,5 +42,5 @@ items:
         targetPort: {{ .Values.service.uiPort }}
     selector:
       app: jaeger
-    type: LoadBalancer
+    type: {{ .Values.service.type }}
```

Comment:
Same as above, should use a different property, e.g. `.Values.service.uiType` set to `LoadBalancer`.